pegasus/sites.v3/code.org/public/yourschoolteststaging.haml

```diff
@@ -0,0 +1,67 @@
+---
+title: Test staging yourschool
+theme: responsive
+social:
+  "og:title": "Does your school teach computer science?"
+  "og:description": "Expand computer science at your school or district. Join the thousands of schools who have already incorporated high quality computer science education into their curriculum and provide opportunities for the students in your local area."
+---
+
+%link{:rel=>'stylesheet', :type=>'text/css', :href=>'/css/yourschool.css'}
+
+- require 'country_codes'
+- require 'state_abbr'
+- require 'census_helper'
+
+- census_announcement = DCDO.get('census_announcement', nil)
+- census_announcement ||= {}
+- census_announcement['user_name'] = request.params["name"] if request.params["name"]
+- census_announcement['user_email'] = request.params["email"] if request.params["email"]
+- census_announcement['is_teacher'] = 'true' if request.params["isTeacher"] == 'true'
+- census_announcement['school_country'] = request.params["country"] if request.params["country"]
+- census_announcement['school_id'] = request.params["schoolId"] if request.params["schoolId"]
+- census_announcement['school_type'] = request.params["schoolType"].capitalize if request.params["schoolType"]
+- census_announcement['school_name'] = request.params["schoolName"] if request.params["schoolName"]
+- census_announcement['school_state'] = request.params["state"] if request.params["state"]
+- census_announcement['school_zip'] = request.params["zip"] if request.params["zip"]
+- census_announcement['school_year'] = current_census_year
+- census_announcement['locale'] = request.locale
+- teacher_application_mode  = DCDO.get("teacher_application_mode", CDO.default_teacher_application_mode)
+- census_announcement['teacher_application_mode'] = teacher_application_mode
+- census_announcement['tileset'] = 'censustiles-test-staging'
```

Comment:
This is hard-coded to `censustiles-test-staging`

Reply:
Do we want this to be `censustiles-dev` as that's what already exists? Or do we want to make a new tileset?